README.md

```diff
@@ -20,15 +26,47 @@ Simply run:
 $ inspec plugin install train-winrm
 ```
 
-You can then run:
+You can then run, using Chef InSpec as an example:
+
+```
+you@home $ inspec some-profile winrm://someuser@somehost --password somepassword
+```
 
+From Ruby code, you may use this plugin as follows:
 ```
-TODO - example
+require 'train'
+transport = Train.create('winrm',
+  host: '1.2.3.4', user: 'Administrator', password: '...', ssl: true, self_signed: true)
```

Comment:
maybe indent this to be idiomatic and readable for the user?

Reply:
Done